config/phpcs/ZooRoyal/ruleset.xml

```diff
@@ -5,18 +5,19 @@
     </description>
 
     <arg name="report" value="full"/>
-    <arg name="report-checkstyle" value="/tmp/checkstyle"/>
+    <arg name="report-checkstyle" value="../../../tmp/checkstyle"/>
```

Comment:
Ich störe mich ein wenig an den `../../../`.

AFAIK kann phpcs`<config>`-Knoten in seinen xml rulesets referenzieren. 

So könnte man aus 
```xml
<arg name="report-checkstyle" value="../../../tmp/checkstyle">
```
ein 
```xml 
<config name="documentRoot" value="../../..">
<arg name="report-checkstyle" value="{$documentRoot}/tmp/checkstyle">
``` 
machen.

Reply:
NVM. Hatte ich falsch im Kopf, schade! 😢 
```
Running PHP Code Sniffer
ERROR: The specified checkstyle report file path "{$documentRoot}/tmp/checkstyle" points to a non-existent directory
```

Reply:
Schade ;/